src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/MicrosoftIdentityPlatform/MicrosoftIdentityPlatformApplicationManager.cs

```diff
@@ -184,10 +184,19 @@ public class MicrosoftIdentityPlatformApplicationManager
             return tenant;
         }
 
-        internal async Task<bool> UpdateApplication(TokenCredential tokenCredential, ApplicationParameters? reconciledApplicationParameters, ProvisioningToolOptions toolOptions)
+        internal async Task<JsonResponse> UpdateApplication(
+            TokenCredential tokenCredential,
+            ApplicationParameters? reconciledApplicationParameters,
+            ProvisioningToolOptions toolOptions,
+            string commandName)
```

Comment:
Is this a breaking change from a VS perspective? would older VS versions fail if they use the new tool version?

Reply:
Not a breaking change from VS perspective, these changes shouldn't cause any problems for older VS versions